javascript/GridFieldExtensions.js

```diff
@@ -195,7 +195,12 @@
 
 				tmpl.cache[this[0].id + "ss-gridfield-add-inline-template"] = tmpl(row.html());
 
-				this.find("tbody:first").append(tmpl(this[0].id + "ss-gridfield-add-inline-template", { num: num }));
+				if ($(button)?.data("prepend-rows")) {
+					this.find("tbody:first").prepend(tmpl(this[0].id + "ss-gridfield-add-inline-template", { num: num }));
+				} else {
+					this.find("tbody:first").append(tmpl(this[0].id + "ss-gridfield-add-inline-template", { num: num }));
+				}
```

Comment:
```suggestion
        const newItem = tmpl(this[0].id + "ss-gridfield-add-inline-template", { num: num });
				if ($(button)?.data("prepend-rows")) {
					this.find("tbody:first").prepend(newItem);
				} else {
					this.find("tbody:first").append(newItem);
				}
```
Just a small nitpick now that I'm looking at this again.